staging/src/kubevirt.io/api/instancetype/v1alpha2/types.go

```diff
@@ -153,6 +152,13 @@ type MemoryInstancetype struct {
 	// Optionally enables the use of hugepages for the VirtualMachineInstance instead of regular memory.
 	// +optional
 	Hugepages *v1.Hugepages `json:"hugepages,omitempty"`
+	// OvercommitPercent is the percentage of the guest memory which will be overcommitted.
+	// This means that the VMIs parent pod (virt-launcher) will request less
+	// physical memory by a factor specified by the OvercommitPercent.
+	// Overcommits can lead to memory exhaustion, which in turn can lead to crashes. Use carefully.
+	// Defaults to 0
+	//+optional
```

Comment:
Missing space, it is present in all the other occurrences.
```suggestion
	// +optional
```